products/vault_selinux/vault.fc

```diff
@@ -2,3 +2,4 @@
 /opt/vault(/.*)?		gen_context(system_u:object_r:vault_sys_content_t,s0)
 /etc/vault\.d(/.*)?		gen_context(system_u:object_r:vault_conf_t,s0)
 /var/log/vault(/.*)?		gen_context(system_u:object_r:vault_log_t,s0)
+/var/log/vault_audit_log	gen_context(system_u:object_r:vault_log_t,s0)
```

Comment:
wouldn't this be `/var/log/vault_audit\.log`? Do you have a rotation policy for your those audit logs? Depending on what's being used, something like:
```
/var/log/vault_audit\.log(\..*)?	gen_context(system_u:object_r:vault_log_t,s0)
```
Might be needed to also target any rotated files.

Reply:
Oh, yeah, `.log`, not `_log`...  Not sure where I saw `_log`.

Maybe should just be `/var/log/vault(.*)`, or drop this and change `/var/log/vault(/.*)?` to `/var/log/vault(.*)?`

Reply:
I think using `/var/log/vault(.*)?` would be fine.